apps/cricket-score-frontend/src/components/MessageBanner.tsx

```
@@ -0,0 +1,23 @@
+  const [isVisible, setIsVisible] = useState(true);
+
+  if (!isVisible) return null;
+
+  return (
+    <div className="alert alert-info mb-6">
+      <div className="flex justify-between items-center relative">
+        <p className="text-foreground">{message}</p>
+        <button
+          onClick={() => setIsVisible(false)}
+          className="btn btn-ghost w-7 h-7 p-0 rounded-full"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)